src/dev/ci_setup/setup.sh

```diff
@@ -10,6 +10,17 @@ echo " -- PARENT_DIR='$PARENT_DIR'"
 echo " -- KIBANA_PKG_BRANCH='$KIBANA_PKG_BRANCH'"
 echo " -- TEST_ES_SNAPSHOT_VERSION='$TEST_ES_SNAPSHOT_VERSION'"
 
+###
+### copy .bazelrc-ci into $HOME/.bazelrc
+###
+cp "src/dev/ci_setup/.bazelrc-ci" "$HOME/.bazelrc";
+
+###
+### generate .bazelrc-ci.auth into "$HOME/.bazelrc-ci.auth";
+###
+echo "# Generated by src/dev/ci_setup/setup.sh" > "$HOME/.bazelrc-ci.auth"
+echo "build --remote_header=x-buildbuddy-api-key=$KIBANA_BUILDBUDDY_CI_API_KEY" >> "$HOME/.bazelrc-ci.auth"
```

Comment:
How about dropping the auth file altogether?

```suggestion
echo "# Appended by src/dev/ci_setup/setup.sh" > "$HOME/.bazelrc"
echo "build --remote_header=x-buildbuddy-api-key=$KIBANA_BUILDBUDDY_CI_API_KEY" >> "$HOME/.bazelrc"
```

Reply:
Let's try that